create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+
+    ```bash
+    export TF_VAR_github_owner="<github_username>"                              # should be pretty self explanatory
+    export TF_VAR_github_token="<github_personal_access_token>"                 # should be pretty self explanatory
+    export TF_VAR_github_repository_name="<git_repository_name>"                # the desired git repository name
+    export TF_VAR_github_repository_branch="<git_repository_branch>"            # branch name to be created
+    export TF_VAR_github_repository_target_path="<flux_cd_sync_target_path>"    # the K8S manifests used to provision Flux CD will be stored in this path
+    ```
+2. Let's go next with setting up the required stuff on the [DigitalOcean](https://cloud.digitalocean.com) side by creating the necessary tokens first. One is needed for creating/managing the DOKS cluster and another one for [DO Spaces](https://cloud.digitalocean.com/spaces) (S3-compatibe object storage). The latter is needed for storing the Terraform state file.
+
+    From your DigitalOcean account go to the [API](https://cloud.digitalocean.com/account/api) section in order to generate the tokens. 
+    
+    Create a personal access token first by clicking the `Generate New Token` button:
+
+    ![Applications & API Key](content/img/pt_gen_1st.png)
+
+    then make sure to give it a proper name and that the `Write` scope is checked:
+
+    ![Applications & API Key](content/img/pt_gen_2nd.png)
+
+    In the end we should have something like this:
+
+    ![Applications & API Key](content/img/api_access_key.png)
+
+    Copy the token value and save it in a local environment variable as we will need it later on (make sure to replace the `<>` placeholder):
+
+    ```bash
+    export TF_VAR_do_token="<your_do_personal_access_token>"
+    ```
+    Next we navigate to `Spaces access keys` and click the `Generate New Key` button:
+
+    ![Spaces Key](content/img/spk_gen_1st.png)
+
+    then give it a proper name and mark it as done:
+
+    ![Spaces Key](content/img/spk_gen_2nd.png)
+
+    In the end we should have something like this:
+
+    ![Spaces Key](content/img/spaces_key.png)
+
+    Copy the key and secret value and save each in a local environment variable for later use (make sure to replace the `<>` placeholder):
+
+    ```bash
+    export DO_SPACES_ACCESS_KEY="<your_do_spaces_access_key>"
+    export DO_SPACES_SECRET_KEY="<your_do_spaces_secret_key>"
+    ```
+3. Going further we have to create a [DO Space](https://cloud.digitalocean.com/spaces) for storing the Terraform state file. Go ahead to your DigitalOcean account panel and click on the `Create` button (the green one) from the upper right corner. Select a region closer to you then make sure that `Restrict file listing` is checked and finally give it a proper name. The required steps are highlighted below:
```

Comment:
```suggestion
3. Create a [DO Space](https://cloud.digitalocean.com/spaces) for storing the Terraform state file. Go ahead to your DigitalOcean account panel and click on the `Create` button (the green one) from the upper right corner. Select a region closer to you then make sure that `Restrict file listing` is checked and finally give it a proper name. The required steps are highlighted below:
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.